VisualizationBase/src/overlays/HighlightOverlay.cpp

```diff
@@ -34,7 +34,7 @@
 
 using namespace Visualization;
 
-namespace InformationScripting {
+namespace Visualization {
```

Comment:
All includes from the same plugin, should use relative paths.

Please look a few lines above this one and make the paths relative to the current file.